File: totosuki/ABC/C/346.py
N, K = map(int, input().split())
A = list(map(int, input().split()))
A = list(set(A))
A.sort()
B = []

for i in range(len(A)):
  if A[i] > K:
    break
  else:
    B.append(A[i])

sm = (1 + K) * K // 2

print(sm - sum(B))

File: OCH_Tao/ABC/A/346.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input[0], 10);
const A = input[1].split(" ").map(x => parseInt(x, 10));
let L = [];
for (let i = 0; i < N - 1; i++) {
  L.push(A[i] * A[i + 1]);
}
console.log(L.join(" "));